recording/server.conf.in

```diff
@@ -109,3 +109,9 @@
 
 # The extension of the file for audio and video recordings.
 #extensionvideo = .webm
+
+[recording]
+# Browser to use for recordings.
+# Allowed values: firefox, chromium
```

Comment:
Although technically it is indeed Chromium I would use `chrome` instead for consistency on the name used by WebDriver.

Reply:
The current implementation specifically requires `chromium`: https://github.com/nextcloud/spreed/pull/9174/files#diff-9ce0e5f38a046c4e6bb945a145850bd7cbd9c22f4f9fb8f82fb3597adcfbc258R273

One could simply remove such `options.binary_location`, and it would all work just fine, but with Google Chrome instead.

Wouldn't it be problematic to call it `chrome`, and then have Google Chrome support added at a latter date?

Reply:
> The current implementation specifically requires `chromium`: https://github.com/nextcloud/spreed/pull/9174/files#diff-9ce0e5f38a046c4e6bb945a145850bd7cbd9c22f4f9fb8f82fb3597adcfbc258R273
>
> One could simply remove such `options.binary_location`, and it would all work just fine, but with Google Chrome instead.

I would prefer if the binary was not hardcoded in the code. And actually I am surprised that it is needed, I would have expected Chromium to be automatically found (provided `chromium-chromedriver` is installed, which your Dockerfile does) even if Selenium/Webdriver is looking for Google Chrome :thinking:

> Wouldn't it be problematic to call it `chrome`, and then have Google Chrome support added at a latter date?

I was actually thinking of `chrome` as a generic name that includes both `chromium` and `chrome` (even if Chrome is actually Chromium, but not the other way around), just for consistency with WebDriver names; I think that `chromium` is used in WebDriver as a root for both Chrome and Edge, not to represent the Chromium browser, and that there is no actual difference between Google Chrome or pure Chromium (both seen as `chrome`) from the point of view of WebDriver, but I could be wrong.

And as a side note, I would not add support for Google Chrome if a FOSS Chromium can be used instead ;-) (unfortunately official Selenium Docker images use Google Chrome rather than Chromium if I am not mistaken, so that is why the helper script of Talkbuchet uses Chrome :-( ).

Reply:
> The binary path is not needed. `chromium-chromedriver` Is used indifferently for Google Chrome or the Chromium web browser, with a preference for Google Chrome for whatever reason. I'd Google Chrome _and_ Chromium installed, reason why I added the binary path.

Ah, I see. While we could make the binary path configurable in the ini file in this case I think it is not worth the trouble (even if I love to be able to configure everything ;-) ). I think that having both browsers installed is not a common setup, and I would also expect the recording server to be typically deployed in an "isolated" environment (inside a container, or in a dedicated server machine), where only the desired browser would be present.

> Will then rename everything from Chromium to Chrome. And leave a note on the ini file (or somewhere else?) that Google Chrome or Chromium will be used, in that order of preference.

Perfect! Yes, I think the ini file is fine, as this is what sysadmins would read rather than the code.